packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -64,7 +64,7 @@ export default function useTabNav() {
 					.focus();
 			}
 			// If we don't have any section blocks, focus the section root.
-			else {
+			else if ( sectionRootClientId ) {
```

Comment:
Should we provide a fallback here? However, I'm not sure what's the best one for Zoom Out mode.

Reply:
@getdave - is the intention that `getSectionRootClientId()` will _always_ return something? My question is, should we handle a fallback in case there isn't a section root, or is the bug that a sectionRoot should always be defined?

Reply:
Well, technically, it does; it returns an empty string, "," which in the editor means the main root. But it's only an internal marker, has no representation in Canvas, and can't be focused.

Reply:
In terms of fallback I'd just allow it to focus the canvas if the section root is an empty string.

Reply:
Updated in [8920245](https://github.com/WordPress/gutenberg/pull/67102/commits/8920245354ba9adfdbcd3a90da227ec8390c4d0a).